### PR TITLE
[11.0][ADD] agreement_helpdesk module

### PR DESCRIPTION
**Pull request by bodedra**
Issue reference https://github.com/ursais/osi-addons/issues/15

### CODE REVIEW

File: agreement_helpdesk/README.rst [hunk 1]
```diff
@@ -0,0 +1,34 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+   :alt: License: LGPL-3
+
+=======================
+Helpdesk Ticket to Task
```
Comment (max3903) [inline agreement_helpdesk/README.rst]: `Agreement / Helpdesk Ticket`

File: agreement_helpdesk/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Copyright (C) 2018 - TODAY, Open Source Integrators
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    "name": "Helpdesk ticket to an agreement",
+    "summary": "Link a helpdesk ticket to an agreement",
+    "version": "11.0.1.0.0",
+    "license": "LGPL-3",
+    "author": "Open Source Integrators, Odoo Community Association (OCA)",
```
Comment (max3903) [inline agreement_helpdesk/__manifest__.py]: No need for the OCA here.

**Review by max3903 — CHANGES_REQUESTED** [review]

**Comment by bodedra** [comment]
@max3903 comments attended!

**Review by max3903 — APPROVED** [review]

**Comment by bodedra** [comment]
@max3903 I will update these PR once https://github.com/OCA/contract/pull/229 PR will be merge.

**Pull request merged**